inspec.gemspec

```diff
@@ -15,38 +15,15 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = "~> 2.4"
 
-  # the gemfile and gemspec are necessary for appbundler so don't remove it
-  spec.files = %w{Gemfile inspec.gemspec README.md LICENSE} + Dir.glob(
-    "{bin,lib,etc}/**/*", File::FNM_DOTMATCH
-  ).reject { |f| File.directory?(f) }
+  # ONLY the aws/azure/gcp files. The rest will come in from inspec-core
+  # the gemspec is necessary for appbundler so don't remove it
+  spec.files =
+    Dir.glob("{{lib,etc}/**/*,inspec.gemspec}")
+      .grep(/aws|azure|gcp|gemspec/)
+      .reject { |f| File.directory?(f) }
 
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
-    .reject { |f| File.directory?(f) || f =~ %r{lib/plugins/.*/test/} }
-
-  # Implementation dependencies
-  spec.add_dependency "chef-telemetry", "~> 1.0"
-  spec.add_dependency "license-acceptance", ">= 0.2.13", "< 2.0"
-  spec.add_dependency "thor", ">= 0.20", "< 2.0"
-  spec.add_dependency "json-schema", "~> 2.8"
-  spec.add_dependency "method_source", "~> 0.8"
-  spec.add_dependency "rubyzip", "~> 1.2", ">= 1.2.2"
-  spec.add_dependency "rspec", "~> 3.9"
-  spec.add_dependency "rspec-its", "~> 1.2"
-  spec.add_dependency "pry", "~> 0"
-  spec.add_dependency "hashie", "~> 3.4"
-  spec.add_dependency "mixlib-log"
-  spec.add_dependency "sslshake", "~> 1.2"
-  spec.add_dependency "parallel", "~> 1.9"
-  spec.add_dependency "faraday", ">=0.9.0"
-  spec.add_dependency "tty-table", "~> 0.10"
-  spec.add_dependency "tty-prompt", "~> 0.17"
-  spec.add_dependency "tomlrb", "~> 1.2"
-  spec.add_dependency "addressable", "~> 2.4"
-  spec.add_dependency "parslet", "~> 1.5"
-  spec.add_dependency "semverse"
-  spec.add_dependency "htmlentities"
-  spec.add_dependency "multipart-post"
-  spec.add_dependency "term-ansicolor"
+  spec.add_dependency "inspec-core", "= #{Inspec::VERSION}"
+  spec.add_dependency "inspec-bin",  "= #{Inspec::VERSION}"
```

Comment:
We're currently setup to do this dep in the other direction, with `inspec-bin` depending on `inspec`. We need that so the the `inspec` library (which is free to use) may be used by the community without requiring `inspec-bin` (which is license-encumbered). That was part of our commitment to the community in the business model change.

Reply:
I thought that was inspec-core now?

Otherwise, what's the point of inspec-core?

Reply:
inspec-core is a lighter-weight version of the library that does not contain anything that needs to be compiled, nor any of the "heavy" plugins such as AWS, Azure, etc. Along with the library, we provide a licensed binstub gem, inspec-core-bin.